### PR TITLE
[example] add testing code for speech command dataset

### PR DESCRIPTION
**Pull request by jingyonghou**

### CODE REVIEW

**Comment by jingyonghou** [comment]
image

File: kws/bin/test.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2021 Binbin Zhang(binbzha@qq.com)
```
Comment (jingyonghou) [inline kws/bin/test.py:1]: this code is original from score.py, so I don't change the copyright infomation

**Review by robin1001 — APPROVED** [review]

File: kws/bin/test.py (hunk 1 above)
Comment (robin1001) [reply kws/bin/test.py:1]: better to name this file to compute_accuracy.py, test.py is ambiguous.

**Review by robin1001 — CHANGES_REQUESTED** [review]
please rename the test.py

File: examples/speechcommand_v1/s0/run.sh [hunk 2]
```diff
@@ -79,3 +79,30 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
...
+  python kws/bin/average_model.py \
+    --dst_model $score_checkpoint \
+    --src_path $dir  \
+    --num ${num_average} \
+    --val_best
+
+  # Testing
+  result_dir=$dir/test_$(basename $score_checkpoint)
+  mkdir -p $result_dir
+  python kws/bin/test.py --gpu 3 \
```
Comment (robin1001) [inline examples/speechcommand_v1/s0/run.sh:94]: compute_accuracy.py

**Review by robin1001 — APPROVED** [review]

**Pull request merged**